packages/lib/EventManager.ts

```diff
@@ -588,6 +588,7 @@ export default class EventManager {
     bookingReferences: PartialReference[];
     isBookingInRecurringSeries?: boolean;
   }) {
+    const log = logger.getSubLogger({ prefix: [`[deleteEventsAndMeetings]: ${event?.uid}`] });
```

Comment:
Add the event uid to the `deleteEventsAndMeetings` logs so we can point to the specific logs for a booking